src/main/resources/config/application-prod.yml

```diff
@@ -145,6 +145,7 @@ sentry:
 application:
   name: oncokb
   profile: prod
+  base-url: 'https://www.oncokb.org'
```

Comment:
When we roll our this change, we need to update the config of MSK and EU instances to correct url